swagger.yaml

```diff
@@ -3980,6 +3980,7 @@ definitions:
       - 'DUAL_STACK_VIPS'
       - 'CLUSTER_MANAGED_NETWORKING'
       - 'USER_MANAGED_NETWORKING'
+      - 'MINIMAL_ISO'
```

Comment:
Where do we have the check in UpdateInfraEnv()?  It should be there, unrelated to the cluster.

Reply:
Here https://github.com/openshift/assisted-service/blob/master/internal/bminventory/inventory.go#L4472

Reply:
Please add

Reply:
Updated